### ISSUE TITLE
[Retail] PvP Talent Raise Abomination still showing as Army of the Dead in Pvp

### ISSUE THREAD

**Issue by Kealke** [Hekili/hekili#2115]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### WoW Version

Retail (Dragonflight)

### Describe the Issue

Theres not an error occuring rather the icon of AOTD appearing instead of Raise Abomination in Pvp scenario's

### How to Reproduce

1. Enter game as Unholy Death Knight
2. Select Pvp talent Raise Abomination
3. Join pvp match

### Player Information (Link)

```Text
build: v10.0.2-2.0.3
level: 66 (66)
class: DEATHKNIGHT
spec: unholy

talents: abomination_limb
    acclimation = 1/1
    all_will_serve = 1/1
    antimagic_barrier = 1/1
    antimagic_shell = 1/1
    antimagic_zone = 1/1
    apocalypse = 1/1
    army_of_the_damned = 1/1
    army_of_the_dead = 1/1
    asphyxiate = 1/1
    brittle = 1/1
    chains_of_ice = 1/1
    cleaving_strikes = 1/1
    coil_of_devastation = 1/1
    coldthirst = 1/1
    dark_transformation = 1/1
    death_strike = 1/1
    ebon_fever = 1/1
    enfeeble = 1/1
    epidemic = 1/1
    eternal_agony = 1/1
    festering_strike = 1/1
    gloom_ward = 1/1
    icebound_fortitude = 1/1
    improved_death_coil = 2/2
    improved_death_strike = 1/1
    improved_festering_strike = 2/2
    infected_claws = 1/1
    merciless_strikes = 1/1
    might_of_thassarian = 1/1
    mind_freeze = 1/1
    morbidity = 2/2
    outbreak = 1/1
    plaguebringer = 1/1
    raise_dead = 1/1
    raise_dead_2 = 1/1
    replenishing_wounds = 1/1
    rune_mastery = 2/2
    runic_attenuation = 1/1
    scourge_strike = 1/1
    summon_gargoyle = 1/1
    superstrain = 1/1
    unholy_aura = 1/2
    unholy_blight = 1/1
    unholy_command = 2/2
    unholy_endurance = 1/1
    unholy_ground = 1/1
    veteran_of_the_third_war = 2/2
    will_of_the_necropolis = 2/2
    wraith_walk = 1/1

pvptalents: necrotic_aura
   necrotic_wounds
   raise_abomination

covenant: none

conduits: none

soulbinds: []

sets: algethar_puzzle_box = 1

gear: algethar_puzzle_box = 1
    breastplate_of_proven_knowledge = 1
    custodians_medallion_of_delusion = 1
    decay_mothers_wrathful_gaze = 1
    frontline_armplates = 1
    frontline_greaves = 1
    frontline_shoulders = 1
    heartlesion_band_of_might = 1
    heartlesion_girdle = 1
    heartlesion_ring_of_might = 1
    hefty_iskaaran_anchor = 1
    potionstained_cloak = 1
    recruits_badge = 1
    wyrmforged_gauntlets = 1
    wyrmforged_sabatons = 1

legendaries: none

itemIDs: 134658, 134685, 134686, 134691, 153724, 153729, 153730, 192007, 192008, 193647, 193663, 193701, 193705, 193712, 197940

settings: aoe = 2
    buffPadding = 0
    combatRefresh = 0.1
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = true
    cycleDebuff = festering_wound
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 8
    damageOnScreen = true
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    enhancedRecheck = true
    gcdSync = true
    maxTime = 10
    nameplateRange = 8
    nameplates = true
    noFeignedCooldown = false
    package = Unholy
    petbased = false
    potion = potion_of_spectral_strength
    regularRefresh = 0.5
    throttleRefresh = false
    throttleTime = false
    disable_iqd_execute = false

toggles: cooldowns = true 
    custom1 = false 
    custom2 = false 
    defensives = false 
    essences = true [overridden]
    interrupts = false 
    mode = automatic 
    potions = false 

keybinds: 
    abomination_limb    = X   [05]
    antimagic_shell     = F1  [05]
    antimagic_zone      = M3  [06]
    any_dnd             = SM3 [06]
    apocalypse          = E   [01]
    army_of_the_dead    = S4  [06]
    asphyxiate          = T   [05]
    chains_of_ice       = S3  [06]
    clawing_shadows     = 2   [01]
    dark_transformation = C   [05]
    death_and_decay     = SM3 [06]
    death_coil          = 3   [01]
    death_grip          = M4  [01]
    death_strike        = Q   [01]
    deaths_advance      = SF  [06]
    deaths_due          = SM3 [06]
    defile              = SM3 [06]
    epidemic            = 5   [01]
    festering_strike    = 1   [01]
    healthstone         = END [05]
    icebound_fortitude  = G   [01]
    lichborne           = F2  [05]
    mind_freeze         = F   [01]
    outbreak            = 4   [01]
    raise_dead          = S5  [06]
    scourge_strike      = 2   [01]
    summon_gargoyle     = S2  [06]
    unholy_blight       = R   [01]
    wound_spender       = 2   [01]
    wraith_walk         = SM4 [06]

warnings: none
```

### Error Messages (Link)

```Text
No errors
```

### Contact Information

Kealke#4410

**Comment by Hekili** [comment]
Could you provide a snapshot taken when AOTD's icon is showing in PvP?

Does the keybind show correctly for it?

A snapshot is required to diagnose this issue.

## Snapshot

Please supply a Snapshot of the addon's decision-making when you are seeing this issue in-game. This is not a screenshot. To generate a Snapshot, please complete the following steps.

 * Log into your WoW character.
 * Recreate the issue you are reporting (i.e., generate 5 Holy Power).
 * When you see the recommendation you disagree with, press ALT-SHIFT-P to Pause and Snapshot (or use ALT-SHIFT-[ to Snapshot without pausing).
   * You can change these keybindings in /hekili > Toggles if needed.
   * If you take the snapshot at a different time, it will not be useful and you'll be asked to submit a new one.
 * Type /hekili and press Enter.
 * Open the Snapshots section on the left side.
 * Select the snapshot that shows the recommendation that you disagree with.
   * Snapshots are labeled with your specialization, the display in question (usually, you'll choose Primary), and the recommendations the addon made with their timings.
 * Click in the Export Snapshot box, press CTRL+A to select all text including hidden text and press CTRL+C to copy.
 * Open [https://pastebin.com](https://github.com/Hekili/hekili/issues/Pastebin), use CTRL+V to paste the snapshot text, then click Create New Paste.
 * Provide the Pastebin ***link*** in a new comment.

This step is essential, as most issues are related to specific classes, specializations, gear, talent choices, or other game systems.  If you do not provide this information, I cannot triage your problem.

**Comment by ilantukh** [comment]
Hi. I have the same problem, here is my snapshot: https://pastebin.com/uWm6zxgM. 

It recommends me AOTD ("Войско мертвых" in Russian) on the BG and correctly shows the keybind for it (Alt+1), but it's replaced with Raise Abomination, and if I use Raise Abomination, Hekili keeps recommending me to use AOTD.

In the snapshot, it says that I have no pvp talents. but I have raise abomination, necrotic aura and necrotic wounds.

I've noticed that running /reload inside BG or arena fixes this problem. It also fixes itself after some time, maybe it's caused by entering and leaving combat, but I'm not 100% sure.

**Comment by Hekili** [comment]
This will be fixed in 2.0.9e.

**Comment by phatzz** [comment]
This doesn't seem to be fixed when in arena, but is fixed when hitting a pvp dummy.
also i cant enable/disable or seem to add "raise_abomination" in the addon.
